game/Board.ts

```diff
@@ -135,10 +140,55 @@ export class Board {
         return this.t[l.r][l.c];
     }
 
+    /*
+     * Getter for Tile and all of its neighbors at a location.
+     */
+    public getNeighborhood(l: Location): Tile[] {
```

Comment:
this works for now 

in future, https://stackoverflow.com/questions/51383031/slice-section-of-two-dimensional-array-in-javascript could be cleaner and maybe faster bc over pointers

Reply:
yeah not using this func at all oops